<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>habitantEditDialog</class>
 <widget class="QDialog" name="habitantEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>313</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>264</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>56</width>
             <height>15</height>
            </size>
           </property>
           <property name="text">
            <string>Фамилия</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lnameEdit">
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fnameEdit">
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Отчество</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="patroEdit">
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Дата рождения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="bdateEdit">
           <property name="maximumSize">
            <size>
             <width>110</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Номер студ. билета</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="studentID">
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Курс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="courseBox">
           <property name="maximumSize">
            <size>
             <width>41</width>
             <height>23</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Группа</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="groupEdit">
           <property name="maximumSize">
            <size>
             <width>113</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Комната</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="roomNumBox">
           <property name="maximumSize">
            <size>
             <width>71</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>отмена</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>сохранить</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>habitantEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>habitantEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
